### PR TITLE
codespell: add page

### PR DESCRIPTION
**Pull request by bl-ue**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Long overdue 🙂 https://github.com/tldr-pages/tldr/pull/5004#issuecomment-743435985

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages/common/codespell.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Check for typos in all text files in the current directory, recursively:
+
+`codespell`
+
+- Correct all typos found in-place:
+
+`codespell --write-changes`
+
+- Skip files with names that match the specified pattern (accepts a comma-separated list of globs):
```
Comment (marchersimon) [inline pages/common/codespell.md:14]: Is there maybe another word for `glob`? I didn't know what a glob is until now and I think there are many others. (That's just my experience, maybe it is more common than I think). Maybe  `accepts a comma-seperated list of patterns using wildcards`?

Comment (bl-ue) [reply pages/common/codespell.md:14]: Oh, yes, definitely. I didn't know that 🙂.  How about `wildcard patterns` instead of `patterns using wildcards`?

Comment (marchersimon) [reply pages/common/codespell.md:14]: I'd prefer `patterns using wildcards`

Comment (bl-ue) [reply pages/common/codespell.md:14]:
```suggestion
- Skip files with names that match the specified pattern (accepts a comma-separated list of patterns using wildcards):
```

**Review by marchersimon — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Looks good to me.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**